examples/features/encryption/README.md

```diff
@@ -91,6 +90,15 @@ successfully up.
 In mutual TLS (mTLS), the client and the server authenticate each other. gRPC
 allows users to configure mutual TLS at the connection level.
 
+In this example, we use the public/private keys created ahead: 
+
+* "server_cert.pem" contains the server certificate (public key). 
+* "server_key.pem" contains the server private key. 
+* "ca_cert.pem" contains the certificate (certificate authority) that can verify the server's certificate.
```

Comment:
"the certificate (certificate authority)"? Perhaps the certificate of the certificate authority?

Reply:
"the certificate _from_ the certificate authority"?

Reply:
Yes

Reply:
> "the certificate (certificate authority)"? Perhaps the certificate of the certificate authority?

yes